src/winter_onboarding_2021/fleet_management_service/middleware.clj

```diff
@@ -0,0 +1,18 @@
+(ns winter-onboarding-2021.fleet-management-service.middleware
+  (:require [ring.util.response :as response]
+            [clojure.walk :as walk]))
+
+(defn wrap-exception-logging [handler]
```

Comment:
Since this function is doing more than just logging (it's also returning a 500 response) we should change the name to reflect that.

Reply:
I think, yes. I will add it.